gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
wrapper stuff is necessary for travis. travis is using an older version that does not work with the protobuf plugin :(